caltech256.ipynb

```diff
@@ -24,6 +24,13 @@
     "!pip install git+https://github.com/cleanlab/cleanvision.git"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "After you install these packages, you may need to restart your notebook runtime in colab before running the rest of this notebook."
```

Comment:
are you sure this only necessary for colab and not the many other notebook environments out there?

if not, consider deleting: "in colab"

Reply:
deleted